docs/getting_started/publishers/setup_piesocket.md

```diff
@@ -1,10 +1,16 @@
 # Set up PieSocket publisher
 
-This document describes how to set up a publisher using [PieSocket](https://www.piesocket.com/) websocket server API.
+---
+
+**Note:** PieSocket proposes a free-plus plan to Wagtail-live users which consists of the free plan with a daily rate message increased to 1 million. To benefit from it, follow the partner link instead of the regular one.
```

Comment:
```suggestion
**Note:** PieSocket offers a 'free-plus' plan to Wagtail-live users. This is the same as the free plan, but with an increased limit of 1 million messages per day. To benefit from this offer, follow the partner link instead of the regular one.
```